downloader/src/main/kotlin/Main.kt

```diff
@@ -216,9 +234,7 @@ object Main {
                         )
                 )
             } else {
-                // TODO: Allow to specify the VCS revision as a parameter.
-                val vcs = VcsInfo.EMPTY.copy(url = projectUrl!!)
-
+                val vcs = VcsInfo(type = vcsType, url = projectUrl!!, revision = vcsRevision, path = vcsPath)
                 Package.EMPTY.copy(id = dummyId, vcs = vcs, vcsProcessed = vcs)
```

Comment:
Shouldn't it be `vcsProcessed = vcs.normalize()`?

Reply:
Done.